action.js

```diff
@@ -19,23 +83,141 @@ window.onload = function meunWindow() {
     window.onscroll = function () {};
   }
 
-  open.addEventListener('click', () => {
-    div.style.display = 'block';
+  function addModals() {
+    let modalitem = "";
+    projectsArray.forEach((project, i) => {
+      let tags = "";
+      project.tags.forEach((tag) => {
+        tags += `<span>${tag}</span>`;
+      });
```

Comment:
- [ ] Please fix the design for desktop screens. Place technologies, and buttons to the right side after the project description and make sure it matches the design on the template.

### Your design

![image](https://user-images.githubusercontent.com/83803414/162283537-a8cb6e6a-0125-44c9-b9e0-d3dcef884477.png)


### Figma design

![image](https://user-images.githubusercontent.com/83803414/162283582-6a22e37d-ce22-4c89-900a-2f5bb2bab98c.png)